webview-ui/src/components/ViolationIncidentsList.tsx

```diff
@@ -211,14 +150,18 @@ const ViolationIncidentsList: React.FC<ViolationIncidentsListProps> = ({
           style={{ marginBottom: "10px" }}
         >
           <CardHeader
-            // actions={{
-            //   actions: (
-            //     <ViolationActionsDropdown
-            //       violation={violation}
-            //       onGetAllSolutions={onGetAllSolutions}
-            //     />
-            //   ),
-            // }}
+            actions={{
+              actions: (
+                <ViolationActionsDropdown
+                  onGetAllSolutions={() => onGetSolution(violation.incidents ?? [], violation)}
+                  fixMessage={
+                    violation.incidents?.length === 1
+                      ? "Resolve 1 incident within this issue"
+                      : `Resolve the ${violation.incidents?.length ?? 0} incidents within this issue`
+                  }
```

Comment:
IMO "Within this issue"  may be borderline redundant & could do with a shortened version like

```suggestion
  fixMessage={`Resolve ${violation.incidents?.length === 1 ? '1 incident' : `${violation.incidents?.length ?? 0} incidents`}`}
```

Reply:
1. it's a feature :)  I agree the message is a bit verbose but it was requested this way [in the spec ](https://github.com/konveyor/editor-extensions/issues/171). @sjd78  are you OK with shortening the labels? Note that in tree view the labels are shorter because (by default) we cannot pass the number of items in the category (labels for context actions are defined in package.json)
2. in general concatenating strings breaks translation (we are English only now but it may change)

Reply:
Resolving the comment to unblock the merge - we can improve the messages later on.